### ISSUE TITLE
variable doesn't create nice sql

### ISSUE THREAD

**Issue by kylesonaty** [kylesonaty/Stellar#4]
```
var filter = "Kyle";
docs.Query<TestObject>().Where(x => x.Name == filter);
```
This generates bad sql.

**Comment by kylesonaty** [comment]
Closed by 88cedfa5dcdd1df2933599909000abe5aa2b7dd1

**Comment by kylesonaty** [comment]
88cedfa5dcdd1df2933599909000abe5aa2b7dd1
